src/daemon/DaemonManager.cpp

```diff
@@ -72,7 +73,14 @@ bool DaemonManager::start(const QString &flags, NetworkType::Type nettype, const
 
     arguments << "--check-updates" << "disabled";
 
+    // --max-concurrency based on threads available. max: 6
+    int32_t concurrency = QThread::idealThreadCount() / 2;
+    if(concurrency > 6)
```

Comment:
You could use qBound() here, otherwise looking good.

Reply:
Thanks, fixed.